packages/type-safe-api/docs/developer_guides/type-safe-api/integrations.md

```diff
@@ -45,20 +43,26 @@ For example:
     )
     ```
 
-## Lambda Integration
+## Lambda integration
+
+For integrating an API operation with a lambda, use the `Integrations.lambda(yourLambdaFunction)`.
+
+## Mock integration
+
+To mock an API operation, use the `MockIntegrations` class, which is included in your generated infrastructure package. This contains an integration for every response that can be returned by your operations.
 
-For integrating an API operation with a lambda, you can use the `Integrations.lambda(yourLambdaFunction)`.
+!!! info
 
-## Mock Integration
+    If you are using `TypeSafeApi` directly or need to mock a response not defined in your model, use `Integrations.mock` for a less type-safe alternative.
```

Comment:
Maybe let's just say "If you need to mock a response not defined in your model, use `Integrations.mock` for a less type-safe alternative."

Using the `TypeSafeRestApi` construct directly would be a pretty rare case and I don't think we need to call it out in the docs :)